xchange-core/src/main/java/org/knowm/xchange/dto/meta/CurrencyMetaData.java

```diff
@@ -19,15 +19,35 @@
   @JsonProperty("min_withdrawal_amount")
   private final BigDecimal minWithdrawalAmount;
 
+  /** Deposit enabled */
+  @JsonProperty("deposit_enabled")
+  private boolean depositEnabled;
+
+  /** Withdraw enabled */
+  @JsonProperty("withdraw_enabled")
+  private boolean withdrawEnabled;
```

Comment:
I don't like the fact that most exchanges will return false as most APIs do not share this information

maybe introducing an enum would be better?

```
enum WalletsHealth {
    /** You can deposit and withdraw founds from the exchange */
    ONLINE,
    DEPOSITS_DISABLED,
    WITHDRAWALS_DISABLED,
    /** You cannot deposit nor withdraw founds from the exchange */
    OFFLINE,
    /** The exchange does not inform us about the health of this wallet */
    UNKNOWN;
}
```

Reply:
@walec51 Makes perfect sense i'll make the changes.